docs/tools/terraform/reference/cookbook/multicloud-postgresql-recipe.rst

```diff
@@ -0,0 +1,71 @@
+Deploy PostgreSQL® services to multiple clouds and regions
+==========================================================
+
+There are many reasons why businesses need to distribute their databases geographically. One of the primary reason is data residency/regulation which mandates user data to stay within certain regions. 
+This example shows a use case where a sysadmin needs to setup three highly-available PostgreSQL databases in three regions and across different cloud providers for data regulation and compliance.
```

Comment:
As a reader, I'd appreciate if what we going to do will be a bit more "scannable", here it is easier to see what we do as a list - 
In this example we show a setup where to follow data regulations and compliance we
- create three highly-available PostgreSQL databases [P.S - I believe it is a single DB just replicated across three regions? ]  in three regions and across different cloud providers
- protect data against database deletion


 `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ can help create multiple services programmatically.